src/client/components/pages/editPage/index.tsx

```diff
@@ -33,26 +33,23 @@ const routes: Step[] = [
   { name: 'jobs', label: 'Jobs', description: 'Upload your data' },
 ];
 
-const BreadCrumb = (props: MenuProps) => {
-  console.log();
-  return (
-    <div className="breadCrumb">
-      {routes.map(({ name, label, description }) => (
-        <div className="breadCrumb__item" key={name} onClick={() => props.changeRoute(name)}>
-          <Icons
-            className={classNames('breadCrumb__icons', { 'breadCrumb__icons--selected': props.step === name })}
-            path={name}
-            alt={name}
-          />
-          <div className="breadCrumb__text">
-            <div className="breadCrumb__text-label">{label}</div>
-            <span className="breadCrumb__description">{description}</span>
-          </div>
+const BreadCrumb = (props: MenuProps) => (
+  <div className="breadCrumb">

```

Comment:
I know you did not changes this in this PR but, can you also uptade the BEM pattern here?

`no camelcase in classNames` 

suggestion: `className="breadcrumb"`

Reply:
No entiendo lo que me queres decir.